src/Goteo/Library/Forms/Model/FilterForm.php

```diff
@@ -57,8 +58,8 @@ public function createForm() {
             ->add('predefineddata', ChoiceType::class, [
```

Comment:
@davidbeig what's this field doing? It's not on the SQL table (neither in past migrations) and it clearly doesn't get saved, even if I remove the `mapped` property. If it's not being used, I propose to remove it from the form.

Reply:
Nevermind, I've just seen that it modifies the start and end dates.